spec/unit/package_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 module Omnibus
   describe Package do
-    let(:path) { '/path/to/files/thing.deb' }
+    let(:path) { "#{windows? ? 'C:' : ''}/path/to/files/thing.deb" }
```

Comment:
This seems hacky, and having a ternary operator in an interpolation operation is very difficult to follow.